sc/shared/src/main/scala/sigmastate/eval/GraphBuilding.scala

```diff
@@ -507,6 +507,10 @@ trait GraphBuilding extends SigmaLibrary { IR: IRContext =>
         else
           error(s"The type of $obj is expected to be Collection to select 'size' property", obj.sourceContext.toOption)
 
+      case Select(obj, SBoolean.ToByte, _) if obj.tpe.isBoolean =>
+          val bool = eval(obj.asBoolValue)
```

Comment:
Not sure if this can be done inline like this or if we need to add a class in `transformers` and a new (or existing?) `OpCode` for boolean.toByte

Reply:
I guess opcode is not needed, from investigating MethodCallSerializer